src/display/pattern_helper.js

```diff
@@ -412,6 +413,8 @@ var TilingPattern = (function TilingPatternClosure() {
             var cssColor = Util.makeCssRgb(color[0], color[1], color[2]);
             context.fillStyle = cssColor;
             context.strokeStyle = cssColor;
+            // Set color needed by image masks (fixes issues 3226 and 8741).
+            graphics.current.fillColor = cssColor;
```

Comment:
do we need to do that for `this.current.strokeColor =`? do we need to do the same for PaintType.COLORED case?